packages/js/app/src/components/NavigationTemplate.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { View } from 'react-native';
+
+import Styles from './Styles';
+
+const NavigationTemplate: React.FunctionComponent<{
+    children: JSX.Element;
+}> = ({ children }) => <View style={Styles.navigation}>{children}</View>;
+
+export default NavigationTemplate;
```

Comment:
Instead of using the default header, I find that it's easier to customize if I created my own custom header. This template is just a template of a header for us to put titles, buttons, etc. at the top.